### ISSUE TITLE
Make it easier to consume any serf environment config

### ISSUE THREAD

**Issue by KushalP** [KushalP/serfclient-py#16]
There are two environment variables that Serf understands at the moment and there may be more in the future. Two of these `SERF_RPC_ADDR` and `SERF_RPC_AUTH` can be found here: https://github.com/hashicorp/serf/blob/06076d51e967471ba32b2e2a32c097981bff4670/command/rpc.go

Create a class `EnvironmentConfig` that parses these values and provides sane defaults if they're not set.

**Comment by KushalP** [comment]
Here's an example of how this might be used with `SerfClient`:

```python
environment = EnvironmentConfig()
client = SerfClient(host=environment.host, port=environment.port, rpc_auth=environment.auth_key)
```

**Comment by msabramo** [comment]
Ah, I like that.

**Comment by msabramo** [comment]
See PR #17.

**Comment by msabramo** [comment]
I think this can be closed as #17 was merged.

Want to push a new release to PyPI?
